new-integration-tests/src/test/java/oracle/weblogic/kubernetes/extensions/ImageBuilders.java

```diff
@@ -182,12 +209,69 @@ private boolean createMiiBasicImage(String imageName, String imageTag) {
 
     // build an image using WebLogic Image Tool
     logger.info("Create image {0} using model directory {1}", image, MODEL_DIR);
-    return createMiiImage(
+    //return createMiiImage(
+    return createImage(
         defaultWitParams()
             .modelImageName(imageName)
             .modelImageTag(MII_BASIC_IMAGE_TAG)
             .modelFiles(modelList)
             .modelArchiveFiles(archiveList)
+            .wdtModelOnly(true)
+            .wdtVersion(WDT_VERSION)
+            .env(env)
+            .redirect(true));
+
+  }
+
+  /**
+   * Create image with basic domain wdt model yaml and sample app.
+   * @param imageName name of the image
+   * @param imageTag tag of the image
+   * @return true if image is created successfully
+   */
+  private boolean createWdtBasicImage(String imageName, String imageTag) {
```

Comment:
code is duplicated in createWdtBasicImage, createMiiBasicImage..refactor ...you can have one method which takes more parameters

Reply:
Refactored